index.html

```diff
@@ -16,7 +16,7 @@ <h2>About</h2>
             skill set. I am committed to lifelong learning, staying up to date with industry trends through workshops and online
             courses. Each day, embrace the challenges and rewards of being a software developer, poised to make a significant impact
             in the world of technology. </p>
-        <img src="nice wprk place.jpg" alt = "working place" style="width: 800px">  
+        <img src="nice wprk place.jpg" alt = "working place" style="max-width: 100%">  
```

Comment:
Just noticed that this image is really, really large! 5000px across is huge. Besides constraining the display size with CSS, I'd encourage you to down sample the image itself: Don't serve a higher resolution than you actually need.